### PR TITLE
Sponsors and Team

### PR DESCRIPTION
**Pull request by lori33c**
I'm still working on adding people to the "Meet the Team" section. I also need to improve the photos on the website as some of them appear distorted.

### CODE REVIEW

**Review by 26samaahmed — CHANGES_REQUESTED** [review]
Sponsor Section looks good! The Teams section also looks good but there is a problem with some of the images where some are stretched out. Look at photo for reference. I'm not sure if this is the case on my side only. I also think the font for the team roles is a bit too small so we can make the font bigger and a bit thicker so it stands out more. We can order the members later on where it's director, then team leads, then the rest is alphabetical.

Screenshot 2024-08-05 212735

**Comment by lori33c** [comment]
I fixed the photos and made the font bigger and bolder as well.
Capture

**Comment by lori33c** [comment]
Capture1

**Comment by 26samaahmed** [comment]
Looks really good Lori 👏🏻 I think what's left right now would be making it responsive and adding the rest of the organizers once they fill out the form! We can leave the sponsors empty as is until we have more updates.

File: src/components/TeamMember.svelte [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    export let name;
+    export let role;
+    export let image;
+    export let leaf_image_left;
+    export let leaf_image_right;
+</script>
+
+<div class="text-center p-2.5 m-2">
+    <div class="relative inline-block">
+        <img class="profile-image rounded-full w-52 h-52 z-10 top-0 left-0 transform relative" src={image} alt={name} />
```
Comment (26samaahmed) [inline src/components/TeamMember.svelte]:
```suggestion
        <img class="border-20 border[##000000] border-solid rounded-full w-52 h-52 z-10 top-0 left-0 transform relative" src={image} alt={name} />
```

File: src/components/TeamMember.svelte [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+        <img class="absolute w-32 h-32 z-0 bottom-0 left-0 transform -translate-x-8 translate-y-2" src={leaf_image_left} alt="leaf left" />
+        <img class="absolute w-32 h-32 z-0 bottom-0 right-0 transform translate-x-8 translate-y-2" src={leaf_image_right} alt="leaf right" />
+    </div>
+    <h3 class="mt-2 text-lg font-semibold">{name}</h3>
+    <p class="text-sm text-gray-500">{role}</p>
+</div>
+
+<style>
+    .profile-image {
+      border-width: 20px; /* Adjust border width */
```
Comment (26samaahmed) [inline src/components/TeamMember.svelte]:
```suggestion

```

File: src/components/TeamMember.svelte [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+      border-color: black; /* Border color */
```
Comment (26samaahmed) [inline src/components/TeamMember.svelte]:
```suggestion
```

File: src/components/TeamMember.svelte [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+      border-style: solid; /* Border style */
```
Comment (26samaahmed) [inline src/components/TeamMember.svelte]:
```suggestion
```

**Review by 26samaahmed — CHANGES_REQUESTED** [review]
Some styling can be translated into Tailwind. I added some changes in the TeamMember.svelte file but you can also try to minimize the use of vanilla css in Team.svelte. Here is a good link where you can search for the specific styling https://tailwindcss.com/docs/flex-basis

**Comment by ryan-avancena** [comment]
very good and the page is responsive and looks good to go

only suggestion is that we might want to add some other visuals in the background for the blue "meet our team" section (if it's not too late), looks v good tho!

**Comment by 26samaahmed** [comment]
I re-ordered the our team so its director, leads, then officers are in last name alphabetical order. Look at photo for reference. It looks really good, good job @lori33c !!
Screenshot 2024-08-24 124137

**Pull request merged**